apps/accounts/models.py

```diff
@@ -64,6 +64,7 @@ class JwtToken(TimeStampedModel):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     access_token = models.CharField(max_length=512, blank=False, null=True)
     refresh_token = models.CharField(max_length=512, blank=False, null=True)
+    expires_at = models.PositiveIntegerField(default=86400)
```

Comment:
Shouldn't it be a date field?

Reply:
Also, why do we need this field? Can't we calculate it dynamically?

Reply:
Because it is epoch time and not a date time field. Replied to second comment on channel